### ISSUE TITLE
Displaying Jupyter Notebook Subsection Links

### ISSUE THREAD

**Issue by christian-rli** [OpenEnergyPlatform/tutorial#141]
Have a look at this tutorial: https://openenergy-platform.org/tutorials/jupyter/tutorial_open-data-licenses/ 

Having all those ¶ symbols looks bad. As @stap-m has pointed out in #140 they should be replaced, ideally with this symbol 🔗.

**Comment by jh-RLI** [comment]
I removed my assignment as @johannwagner  has implemented the jupyter notebook view. He probably knows how to adjust this? Reassign me if I can help somehow.

**Comment by jh-RLI** [comment]
One more note, this is about the view in the tutorials app on the oep. Therefore, it would be better to create the issue in the OEP repo!?

**Comment by johannwagner** [comment]
Yup, this is a displaying issue which has nothing to do with the content. +1 for @jh-RLI proposal.
